### ISSUE TITLE
Error image when using Big Admin account 

### ISSUE THREAD

**Issue by akenner** [PECE-project/pece-distro#115]
![Screen Shot 2020-11-11 at 12 09 00 PM](https://user-images.githubusercontent.com/1063084/98851957-fb642900-2424-11eb-98b9-8ed3ecfae33a.png)

This error message appeared when I attempted to create an image artifact using the big admin account on Housing Energy.

**Comment by Honoluluskye** [comment]
Yes, I also got the same error message when I just tried to add a new audio artifact into Research Data Share. Tried it as my individual user and then also as big Admin and both returned the same error.
![Screen Shot 2020-11-12 at 10 20 41 AM](https://user-images.githubusercontent.com/6000846/98980046-fceb2b00-24d0-11eb-9b9d-6415cc082a80.png)
![Screen Shot 2020-11-12 at 10 18 38 AM](https://user-images.githubusercontent.com/6000846/98980055-fe1c5800-24d0-11eb-9121-eb0e4f686c2b.png)

**Comment by Honoluluskye** [comment]
@revagomes

**Comment by revagomes** [comment]
@Honoluluskye , @akenner I've updated PECE v1.29.1 release to add a fix for that. 

Please let me know if it's working as expected now.

**Comment by Honoluluskye** [comment]
I am able to create new artifacts now but I think there's still something buggy with either the tags or the audience fields. I was able to upload when I didn't put anything in the tags or audience. Then I went back to actually add the real info and got this error (attached). Then when I went back to edit, the tags were gone and I removed audience and the rest of the new meta data still saved okay.
![Screen Shot 2020-11-19 at 9 22 25 AM](https://user-images.githubusercontent.com/6000846/99701226-06ccdb00-2a49-11eb-9478-3ff6933886f3.png)
